docs/docsite/rst/community/collection_contributors/collection_release_with_branches.rst

```diff
@@ -22,17 +22,19 @@ Releasing major collection versions
 
 The new version is assumed to be ``X.0.0``.
 
-1. If you are going to release the ``community.general`` and ``community.network`` collections, create new ``backport-X`` and ``needs_backport_to_stable_X`` labels in the corresponding repositories. Copy the styles and descriptions from the corresponding existing labels.
+1. Check the collections for deprecations that are planned for removal in the major release. Use past changelogs or run ``grep -r `X.0.0` plugins/`` in the repository.
```

Comment:
This should only be done once the version in `galaxy.yml` has already been updated to X.0.0 and sanity tests are passing. That step should have made sure that most deprecations have already been handled, and this is an extra step to verify nothing was missed.

Reply:
@felixfontein sorry, I don't have a lot of experience with major releases, deprecations, etc. (i see the typo there btw it should be `collection`, not `collectionS`).
Could you please suggest your wording or elaborate a bit more?

Reply:
I would probably make this 2., and add a 1. before it:
```rst
1. Make sure that ``galaxy.yml`` contains the correct version number ``X.0.0``. If that is not the case, create a PR to update it. This will make sanity tests fail for all deprecations that have to be removed in ``X.0.0``, so this is potentially a lot of work and should have been done weeks before the major release.
```
And then remove

> 6. In the ``stable-X`` branch, ensure that ``galaxy.yml`` contains the correct version number ``X.0.0``. If not, update it.

resp. change it to

> 6. In the ``stable-X`` branch, verify that ``galaxy.yml`` contains the correct version number ``X.0.0``.

Reply:
@felixfontein SGTM, thanks for clarifying, changed as you suggested, PTAL